openpype/tools/pyblish_pype/app.py

```diff
@@ -90,9 +90,8 @@ def show(parent=None):
         install_fonts()
         install_translator(app)
 
-        ctrl = control.Controller()
-
         if self._window is None:
+            ctrl = control.Controller()
```

Comment:
This technically wasn't crucial to this PR but I noticed the controller would only be used if there was no original window so I moved it here solely because I was confused at first thinking the controller was always "cleared" even if the window persisted.